rosdep/python.yaml

```
@@ -1941,6 +1941,13 @@ python-hypothesis:
   gentoo: [dev-python/hypothesis]
   ubuntu:
     '*': [python-hypothesis]
+python-ifaddr:
```

@cottsay do you know why it would be suggesting `python2-ifaddr` for opensuse when the package URL itself is `python-ifaddr`?

---

The `python2-ifaddr` RPM package provides a virtual package called `python-ifaddr`:
```
$ rpm --qf '[%{PROVIDES}\n]' -qp https://download.opensuse.org/distribution/leap/15.2/repo/oss/noarch/python2-ifaddr-0.1.6-lp152.2.1.noarch.rpm
python-ifaddr
python2-ifaddr
```

In Fedora, it was normal for Python packages to provide the "unversioned" python package name to ease the transition away from the unversioned package names. Evidently openSUSE did the same thing.

Two notes here:
1. We should update our openSUSE Leap version target to 15.5, where the Python 2 packages are no longer available so warnings like this won't show up.
2. Why is the key `python-ifaddr` and not `python3-ifaddr`? Shouldn't new keys follow the Ubuntu package name?

---

```suggestion
python3-ifaddr:
```

For new python keys they should be `python3-*` to match the key values which all appear to be python 3. 

Note that this will require it to be re-alphabetized too. And should avoid the opensuse python2 question above too.